wordpress-mobile/WordPress-Android
# WP Stories: remove ids array from stories block structure

Fixes #12732 

Note: builds on top of #12729 since there's related changes in there 

To test:
1. create a Story
2. observe it creates the gb block structure without the `ids` array
3. observe the passed Post content is the same on the remote site.

PR submission checklist:
